sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -1,56 +1,56 @@
 # Azure Cognitive Language Services Conversations client library for .NET
 
-The Question Answering service is a cloud-based API service that lets you create a conversational question-and-answer layer over your existing data. Use it to build a knowledge base by extracting questions and answers from your semi-structured content, including FAQ, manuals, and documents. Answer users’ questions with the best answers from the QnAs in your knowledge base—automatically. Your knowledge base gets smarter, too, as it continually learns from user behavior.
+Conversations (aka LuisVNext) is a cloud-based conversational AI service that applies custom machine-learning intelligence to a user's conversational, natural language text to predict overall meaning, and pull out relevant, detailed information. The service utilizes state-of-the-art technology to create and utilize natively multilingual models, which means that users would be able to train their models in one language but predict in others. 
```

Comment:
"LuisVNext" not only seems formatted incorrect ("LUIS vNext"?) but a term customers probably aren't familiar with nor would they need to know. Maybe instead something like, "Azure Conversations - the new version of Language Understanding (LUIS) - is a cloud-based ..."

Should also say "pulls out ..." for word agreement with "applies custom ...".

```suggestion
Azure Conversations - the new version of Language Understanding (LUIS) - is a cloud-based conversational AI service that applies custom machine-learning intelligence to a user's conversational, natural language text to predict overall meaning; and pulls out relevant, detailed information. The service utilizes state-of-the-art technology to create and utilize natively multilingual models, which means that users would be able to train their models in one language but predict in others. 
```